liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -155,7 +155,7 @@ public static void main(String[] args) {
         try {
             errorLevel = run(args);
         } catch (Throwable e) {
-            System.exit(-1);
+            System.exit(1);
```

Comment:
Changing from -1 to positive 1 is still a non-zero return code.  So this one does not look necessary.